frontend/src/text_resources.js

```diff
@@ -259,6 +262,9 @@ let LOCALIZE = new LocalizedStrings({
           bullet4: "Vous pouvez répondre aux courriels dans l’ordre que vous désirez.",
           bullet5: "Vous êtes responsable de la gestion de votre temps."
         },
+        instructions: {
+          title: "FR Test Instructions"
```

Comment:
Same as my previous comment. Do you want me to translate this one?

Reply:
If you don't mind, please!

Reply:
Done in commit #https://github.com/code-for-canada/project-thundercat/pull/84/commits/f69b1e8aca6b3426cc25b4cf2812d7bcc3f9a1ab